docs/reference/announcements.md

```diff
@@ -4,14 +4,32 @@ title: "Announcements"
 description: "Important announcements including deprecation & removal notices"
 ---
 
-## Upgrading to 8.2
+## Camunda Platform 8.2
+
+Release date: 11th of April 2023
+
+End of maintenance: 8th of October 2024
+
+### Updating to 8.2
```

Comment:
shouldnt this be part of update guide instead?

Reply:
Good catch! I'll move it 😄